app/controllers/concerns/secured.rb

```diff
@@ -6,22 +6,13 @@ module Secured
   end
 
   def authenticate_request!
-    auth_token
-  rescue JWT::VerificationError, JWT::DecodeError
-    render json: { errors: ['Not Authenticated'] }, status: :unauthorized
-  end
-
-  def http_token
-    if request.headers['Authorization'].present?
-      request.headers['Authorization'].split(' ').last
+    @user = System.authentication.current_user(request)
+    unless @user
+      render status: 401
     end
   end
 
-  def auth_token
-    JsonWebToken.verify(http_token)
-  end
-
-  def auth0_id
-    auth_token.first['sub']
+  def user
```

Comment:
What do you think about renaming this method (and the variable) to `current_user`?
Since `user` is a common name for a variable, I can see how it could create subtle bugs if some controller decides to create its own user variable.